File: accidentes/FicheroAccidentes-Java-IvanRC/src/main/java/dto/accidente/AccidenteDto.java
package dto.accidente;

import dto.distrito.DistritoDto;
import dto.lesividad.LesividadDto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.simpleframework.xml.Element;
import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Root(name = "accidente")
public class AccidenteDto {
    @Element(name = "numero_de_expediente")
    String numeroExpediente; // 0

    @Element(name = "fecha")
    String fecha; // 1

    @Element(name = "hora")
    String hora; // 2

    @ElementList(name = "localizacion", inline = true)
    List<String> localizacion; // 3

    @Element(name = "numero_calle")
    String numeroCalle; // 4, en caso de que no sea válido pues un nulo

    @Element(name = "distrito")
    DistritoDto distrito;

    @Element(name = "tipo_de_accidente")
    String tipoAccidente; // 7

    @Element(name = "estado_meteorologico")
    String estadoMeteorologico; // 8

    @Element(name = "tipo_de_vehiculo")
    String tipoDeVehiculo; // 9

    @Element(name = "tipo_de_persona")
    String tipoDePersona; // 10

    @Element(name = "rango_de_edad")
    String rangoEdad; // 11

    @Element(name = "sexo")
    String sexo; // 12

    @Element(name = "lesividad")
    LesividadDto lesividad;

    @Element(name = "longitud")
    String longitud; // 15

    @Element(name = "altitud")
    String altitud; // 16

    @Element(name = "es_positiva_en_alchol")
    String esPositivaEnAlchol; // 17, N == false y S == true

    @Element(name = "es_positiva_en_drogas")
    String esPositivaEnDrogas; // 18, NULL == false y 1 == true
}

File: accidentes/FicheroAccidentes-Kotlin-IvanRC/src/main/kotlin/dto/accidente/AccidenteDto.kt
package dto.accidente

import dto.distrito.DistritoDto
import dto.lesividad.LesividadDto
import org.simpleframework.xml.Element
import org.simpleframework.xml.ElementList
import org.simpleframework.xml.Root

@Root(name = "accidente")
data class AccidenteDto (
    @field:Element(name = "numero_de_expediente")
    @param:Element(name = "numero_de_expediente")
    val numeroExpediente: String, // 0

    @field:Element(name = "fecha")
    @param:Element(name = "fecha")
    val fecha: String, // 1

    @field:Element(name = "hora")
    @param:Element(name = "hora")
    val hora: String, // 2

    @field:ElementList(name = "localizacion", inline = true)
    @param:ElementList(name = "localizacion", inline = true)
    val localizacion: List<String>, // 3

    @field:Element(name = "numero_calle")
    @param:Element(name = "numero_calle")
    val numeroCalle: String, // 4, en caso de que no sea válido pues un nulo

    @field:Element(name = "distrito")
    @param:Element(name = "distrito")
    val distrito: DistritoDto,

    @field:Element(name = "tipo_de_accidente")
    @param:Element(name = "tipo_de_accidente")
    val tipoAccidente: String, // 7

    @field:Element(name = "estado_meteorologico")
    @param:Element(name = "estado_meteorologico")
    val estadoMeteorologico: String, // 8

    @field:Element(name = "tipo_de_vehiculo")
    @param:Element(name = "tipo_de_vehiculo")
    val tipoDeVehiculo: String, // 9

    @field:Element(name = "tipo_de_persona")
    @param:Element(name = "tipo_de_persona")
    val tipoDePersona: String, // 10

    @field:Element(name = "rango_de_edad")
    @param:Element(name = "rango_de_edad")
    val rangoEdad: String, // 11

    @field:Element(name = "sexo")
    @param:Element(name = "sexo")
    val sexo: String, // 12

    @field:Element(name = "lesividad")
    @param:Element(name = "lesividad")
    val lesividad: LesividadDto,

    @field:Element(name = "longitud")
    @param:Element(name = "longitud")
    val longitud: String, // 15

    @field:Element(name = "altitud")
    @param:Element(name = "altitud")
    val altitud: String, // 16

    @field:Element(name = "es_positiva_en_alchol")
    @param:Element(name = "es_positiva_en_alchol")
    val esPositivaEnAlchol: String, // 17, N == false y S == true

    @field:Element(name = "es_positiva_en_drogas")
    @param:Element(name = "es_positiva_en_drogas")
    val esPositivaEnDrogas: String // 18, NULL == false y 1 == true
)